src/Content/Text/BBCode.php

```diff
@@ -1113,7 +871,7 @@ function ($match) use ($simplehtml, $uriid) {
 				}
 
 				$img_str = '<img src="' .
-				self::proxyUrl($match[2], $simplehtml, $uriid) . '"';
+					self::proxyUrl($match[2], $simplehtml, $uriid) . '"';
```

Comment:
New line is superfluous here, it should be combined with line 873.

Reply:
Done